packages/sdk-react/src/core/queries/useProducts.ts

```diff
@@ -87,7 +86,7 @@ export const useInfiniteProducts = (
         paginationToken: pageParam,
       });
     },
-    initialPageParam: '0',
+    initialPageParam: '',
```

Comment:
Why not make `initialPageParam` as `undefined`?

Reply:
Thanks, I will check

Reply:
I checked. Schema requires a string.